backends/tc/tc.def

```diff
@@ -288,6 +292,19 @@ class TCTable {
             }
             tcTable += " action " + defaultMissAction->getName();
         }
+        if (const_entries.size() != 0) {
+            for (auto entry : const_entries) {
+                tcTable += "\n$TC p4template create table/" + pipelineName
+                      + "/" + controlName + "/" + tableName
+                      + " entry";
+                for(auto k : entry.second) {
+                    tcTable += " " + k.first + " " + k.second;
```

Comment:
Is k.second hex digit string? if so, shouldn't it be prefixed with 0x?

Reply:
no not required. The type will be extracted from introspection json file